snooty.toml

```diff
@@ -3,7 +3,8 @@ title = "Java"
 toc_landing_pages = [
     "/fundamentals/connection",
     "/fundamentals/crud",
-    "/fundamentals/builders"
+    "/fundamentals/builders",
+    "/usage-examples",
```

Comment:
Question: Should there be a comma at the end of the last item?

Reply:
Not sure. Although there weren't any errors, I'll remove it just in case.